.github/workflows/continuous-integration.yml

```diff
@@ -1,284 +1,81 @@
-          uses: actions/checkout@v2
-          with:
-            fetch-depth: 0
-        - name: Using Node.js 20.x
-          uses: actions/setup-node@v1
-          with:
-            node-version: 20.x
-        - name: Clean Install
-          run: npm ci
-        - name: Build
-          run: npm run build:release
+jobs:
+  build:
+    name: Build and Run Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Using Node.js 20.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 20.x
+      - name: Clean Install
+        run: npm ci
+      - name: Build
+        run: npm run build:release
 
-    integ-content-share-job-one:
-      name: Content Share Integration Test Suite One
-      runs-on: ubuntu-latest
-      steps:
-        - name: Checkout Package
-          uses: actions/checkout@v2
-          with:
-            fetch-depth: 0
-        - name: Check if needed to run
-          id: test_needed
-          run: |
-            source ${GITHUB_WORKSPACE}/integration/js/script/need-integ-test
-            check_if_integ_tests_required
-            echo "integ_test_required=$requires_integration_test" >> $GITHUB_OUTPUT
-        - name: Setup GitHub Actions Host
-          if:  steps.test_needed.outputs.integ_test_required == 'true'
-          uses: ./.github/actions/setup-integration-test
-          with:
-            sauce-username: ${{ secrets.SAUCE_USERNAME }}
-            sauce-access-key: ${{ secrets.SAUCE_ACCESS_KEY }}
-            aws-role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME_SDK_DEV }}
-            aws-role-session-name: ${{ env.TEST_TYPE }}
-        - name: Run Content Share Integration Test
-          if:  steps.test_needed.outputs.integ_test_required == 'true'
-          run: npm run test:integration-content-share-test-suite-one
+  audio-test:
+    name: Audio Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check if needed to run
+        id: test_needed
+        run: |
+          source ${GITHUB_WORKSPACE}/integration/js/script/need-integ-test
+          check_if_integ_tests_required
+          echo "integ_test_required=$requires_integration_test" >> $GITHUB_OUTPUT
+      - name: Setup GitHub Actions Host
+        if: steps.test_needed.outputs.integ_test_required == 'true'
+        uses: ./.github/actions/setup-integration-test
+        with:
+          aws-role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME_SDK_DEV }}
+          aws-role-session-name: ${{ env.TEST_TYPE }}
+      - name: Run Audio Test
+        if: steps.test_needed.outputs.integ_test_required == 'true'
+        working-directory: ./integration/mocha-tests
+        run: npm run test -- --test-name AudioTest --host local --test-type integration-test --retry 0 --headless true
```

Comment:
No retries?

Reply:
Just for testing :D. WIll remove it in the final PR